airflow/providers/dbt/cloud/hooks/dbt.py

```diff
@@ -413,6 +414,8 @@ def trigger_job_run(
             instead of those configured in dbt Cloud.
         :param schema_override: Optional. Override the destination schema in the configured target for this
             job.
+        :param retry_from_failure: Optional. If set to True, the job will be triggered using the "rerun"
```

Comment:
I thing we should mention this is mutually exclusive with `steps_override, schema_override, or additional_run_config`